server/server.js

```diff
@@ -79,17 +88,26 @@ const server = app.listen(port, () => {
 });
 
 const io = Server.listen(server);
-
 const attachSocketEvents = socketEvents(store);
-
+store.subscribe(
+  () => io.emit('state', store.getState())
+);
 io.on('connection', (socket) => {
-  console.log('connected!');
-  socket.emit('connected', 'hello from the server');
   socket.emit('state', store.getState());
 });
 
 io.on('connection', attachSocketEvents);
-
-store.subscribe(
-  () => io.emit('state', store.getState())
-);
+socketioAuth(io, {
```

Comment:
Sorry, referring to the `authenticate` function. `'authentication'` event is emitted only on login right? So are we only validating auth on login? Where do we check if each subsequent request / action is authorized?